src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -192,6 +191,15 @@ def _deploy_arm_template_core(resource_group_name, deployment_name, template_fil
     else:
         return smc.deployments.create_or_update(resource_group_name, deployment_name, properties)
 
+def _load_json(location):
+    from six.moves.urllib.request import urlopen #pylint: disable=import-error
+    try:
+        txt = urlopen(location).read()
+        json_content = json.loads(txt.decode())
+    except: #pylint: disable=bare-except
+        json_content = get_file_json(location)
+    return json_content
```

Comment:
Should we attempt to load the local file first before attempting the HTTP request?

Motivation for this is this scenario:
(1) I have `mywebsite/mytemplate.json` on disk
(2) There also exists a network resource at `http://mywebsite/mytemplate.json`

(3) Now, I run this command (or whatever it should be):
`az group deployment --template-file-path mywebsite/mytemplate.json`

The file from the network will be loaded right?
Just thinking that what if I didn't know about (2). Now some template I didn't know about would be used right?